network_analysis/find-connected-features-in-utility-network/src/main/resources/terminal_selection_dialog.fxml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<?import javafx.scene.control.ButtonType?>
+<?import javafx.scene.control.Dialog?>
+<?import javafx.scene.control.DialogPane?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.StackPane?>
+<fx:root type="javafx.scene.control.Dialog" xmlns:fx="http://javafx.com/fxml">
+    <dialogPane>
+        <DialogPane>
+            <content>
+                <StackPane>
+                    <ListView fx:id="utilityTerminalListView" maxHeight="100"/>
+                </StackPane>
+            </content>
+            <ButtonType fx:id="cancelButton" buttonData="CANCEL_CLOSE" text="Cancel"/>
```

Comment:
same with cancelButton, can't see it's FXML object instantiation anywhere? Does it need to have an fx:id?